source/developers/plugins/studio-plugins.rst

```diff
@@ -1,6 +1,6 @@
 :orphan:
 
-:is-up-to-date: True
+:is-up-to-date: False
```

Comment:
What's missing to make this up to date?

Reply:
I am working on updating the directory structure used. Will push changes when done.  Thank you.